python/ray/worker.py

```diff
@@ -1843,16 +1843,6 @@ def method(self):
             crashes unexpectedly. The minimum valid value is 0,
             the default is 4 (default), and a value of -1 indicates
             infinite retries.
-        runtime_env (Dict[str, Any]): Specifies the runtime environment for
-            this actor or task and its children. See``runtime_env.py`` for
-            detailed documentation.
-        override_environment_variables (Dict[str, str]): This specifies
-            environment variables to override for the actor or task.  The
-            overrides are propagated to all child actors and tasks.  This
-            is a dictionary mapping variable names to their values.  Existing
-            variables can be overridden, new ones can be created, and an
-            existing variable can be unset by setting it to an empty string.
```

Comment:
Is this intentional?

Reply:
Ah good point, I took it out because these aren't actually supported by @ray.remote(), but I guess the documentation for `.options` comes from this docstring, so let me put it back